build/helm/edgemesh/charts/agent/templates/04-configmap.yaml

```diff
@@ -93,6 +93,9 @@ data:
         {{- if .Values.modules.edgeProxy.enable }}
         enable: {{ .Values.modules.edgeProxy.enable }}
         {{- end }}
+        {{- if .Values.modules.edgeProxy.invertServiceFilter }}
```

Comment:
If you modified helm charts, then you must regenerate edgemesh.tgz with `helm package` command. Otherwise helm check ci will fail

Reply:
rebuilt